docs/source/usecases/homesale-forecast.rst

```diff
@@ -0,0 +1,135 @@
+    </td>
+    <td>
+        <a target="_blank" href="https://github.com/georgia-tech-db/eva/raw/staging/tutorials/16-homesale-forecasting.ipynb"><img src="https://www.tensorflow.org/images/download_logo_32px.png" /> Download notebook</a>
+    </td>
+    </table><br><br>
+    </embed>
+
+Introduction
+------------
+
+In this tutorial, we present how to use :ref:`forecasting models<forecast>` in EvaDB to predict home sale price. EvaDB makes it easy to do time series predictions using its built-in Auto Forecast function.
+
+.. include:: ../shared/evadb.rst
+
+.. include:: ../shared/postgresql.rst
+
+We will assume that the input data is loaded into a ``PostgreSQL`` database. 
+To load the home sales data into your database, see the complete `home sale forecasting notebook on Colab <https://colab.research.google.com/github/georgia-tech-db/eva/blob/staging/tutorials/16-homesale-forecasting.ipynb>`_.
+
+Preview the Home Sales Data
+-------------------------------------------
+
+We use the `raw_sales.csv of the House Property Sales Time Series <https://www.kaggle.com/datasets/htagholdings/property-sales?resource=download>`_ in this usecase. The data contains five columns: postcode, price, bedrooms, datesold, and propertytype.
+
+.. code-block:: sql
+
+   SELECT * FROM postgres_data.home_sales LIMIT 3;
+
+This query previews the data in the home_sales table:
+
+.. code-block:: 
+
+    +---------------------+------------------+---------------------+---------------------+-------------------------+
+    | home_sales.postcode | home_sales.price | home_sales.bedrooms | home_sales.datesold | home_sales.propertytype |
+    |---------------------|------------------|---------------------|---------------------|-------------------------|
+    |                2607 |           525000 |                   4 |          2007-02-07 |                   house |
+    |                2906 |           290000 |                   3 |          2007-02-27 |                   house |
+    |                2905 |           328000 |                   3 |          2007-03-07 |                   house |
+    +---------------------+------------------+---------------------+---------------------+-------------------------+
+
+Train a Home Sale Forecasting Model
+-----------------------------------
+
+Let's next train a time-series forecasting model from the home_sales table using EvaDB's ``CREATE FUNCTION`` query.
+Particularly, we are interested in the price of the properties that have three bedrooms and are in the postcode 2607 area.
+
+.. code-block:: sql
+
+  CREATE FUNCTION IF NOT EXISTS HomeSaleForecast FROM
+    (
+      SELECT propertytype, datesold, price
+      FROM postgres_data.home_sales
+      WHERE bedrooms = 3 AND postcode = 2607
+    )
+  TYPE Forecasting
+  PREDICT 'price'
```

Comment:
Do you think we should explain these parameters here or provide a link to forecast docs?

Reply:
I have a note after the query that links to forecast docs, please check.

Reply:
Missed it!